docs/resources/tag.md

```diff
@@ -37,25 +47,42 @@ resource "snowflake_tag" "tag" {
 
 ### Required
 
-- `database` (String) The database in which to create the tag.
-- `name` (String) Specifies the identifier for the tag; must be unique for the database in which the tag is created.
-- `schema` (String) The schema in which to create the tag.
+- `database` (String) The database in which to create the tag. Due to technical limitations (read more [here](https://github.com/Snowflake-Labs/terraform-provider-snowflake/blob/main/docs/technical-documentation/identifiers_rework_design_decisions.md#known-limitations-and-identifier-recommendations)), avoid using the following characters: `|`, `.`, `(`, `)`, `"`
+- `name` (String) Specifies the identifier for the tag; must be unique for the database in which the tag is created. Due to technical limitations (read more [here](https://github.com/Snowflake-Labs/terraform-provider-snowflake/blob/main/docs/technical-documentation/identifiers_rework_design_decisions.md#known-limitations-and-identifier-recommendations)), avoid using the following characters: `|`, `.`, `(`, `)`, `"`
```

Comment:
I just realized that we loosened the restriction for parentheses but we did not update this function to list the forbidden chars. Let's update it